base/src/commonMain/kotlin/utils/KalugaDate.kt

```diff
@@ -30,102 +32,107 @@ expect class KalugaDateHolder
  * Class describing a point in time
  * Dates are localized according to a [Locale] and relative to a given [TimeZone]
  */
-interface KalugaDate : Comparable<KalugaDate> {
+abstract class KalugaDate : Comparable<KalugaDate> {
```

Comment:
Why the switch?

Reply:
Oh shit, accidentally contains code from https://github.com/splendo/kaluga/pull/625
Reverting

Reply:
https://github.com/splendo/kaluga/commit/4c7e43d74359c19fc5191bfd63be2507b97598e3#commitcomment-97738256